test/unit/project_type/test_git.py

```diff
@@ -247,7 +247,7 @@ def test_slave_param_overrides_returns_expected(self):
                                                                'expected.')
 
     def test_slave_param_overrides_when_get_project_from_master_is_disabled(self):
-        Configuration['get_project_from_master'] = True
+        Configuration['get_project_from_master'] = False
```

Comment:
The test was failing. I introduced the test break in the previous pull, it was an incorrect merge resolution.